common/cosmwasm-smart-contracts/mixnet-contract/src/gateway.rs

```diff
@@ -112,6 +112,21 @@ impl Display for GatewayBond {
     }
 }
 
+#[derive(Clone, Debug, Deserialize, PartialEq, Eq, Serialize, JsonSchema)]
+pub struct GatewayConfigUpdate {
```

Comment:
I don't remember how it works for the FE, but shouldn't we also derive `ts_rs` (and export types with the tool), so that they could more easily pass that type around?

Reply:
Added!